samples/sensor/hts221/sample.yaml

```diff
@@ -1,7 +1,17 @@
 sample:
   name: HTS221 Temperature and Humidity Monitor
+common:
+    tags: sensor
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: false
+      regex:
+        - "Temperature:(.*)C"
+        - "Relative Humidity:(.*)%"
+      fixture: fixture_i2c_hts221 # see documentation
```

Comment:
Can you provide an example on how this should be populated on board side?
For instance, on disco_l475_iot1 that embeds hts221 sensor

Reply:
@erwango:
> Can you provide an example on how this should be populated on board side?

This entirely depends on the way you use this information. Here, the fixture "fixture_i2c_hts221" is just a way of saying that this test case that is dependent on an external device(HTS221 sensor in this case) and it should be pick a device that fulfills this dependency. So, it entirely depends on your test setup on how you design the board selection(pick one board out of multiple in an automation setup) logic based on "fixture" keyboard.

> For instance, on disco_l475_iot1 that embeds hts221 sensor

For the specific board disco_l475_iot1 that embeds hts221 sensor, you can just have another case specified in sample.yaml such as:
```
  sensor.hts221.embedded:
    tags: sensor
    # disco_l475_iot1 has hts221 sensor present internally
    platform_whitelist: disco_l475_iot1
    depends_on: i2c
```